src/district0x/ui/events.cljs

```diff
@@ -752,20 +777,20 @@
 (reg-event-fx
   :district0x.location/set-query
   interceptors
-  (fn [_ args]
-    {:location/set-query args}))
+  (fn [_ [hashroutes? args]]
```

Comment:
To events `:district0x.location/set-query`, `:district0x.location/nav-to`, `:district0x.location/add-to-query` you don't need to pass `hashroutes?` param, because you have db available, you can figure it out directly.

Reply:
silly of me, sorry.